### DISCUSSION TITLE
Is there any way to use DaisyUI with Tailwind CLI (without npm)?

### DISCUSSION THREAD

**Discussion by arqalite** [saadeghi/daisyui#1739] [Q&A] [answered]
Hello,

I'm interested in using DaisyUI, but my current toolchain doesn't use npm. I use the Tailwind CLI and I wondered if there is any way to use DaisyUI with it and avoid downloading npm and all that stuff.

I tried downloading the npm package and pointing to it using "plugins: [require("/home/arqalite/Downloads/daisyui-2.51.5/package")]", but it complains that it can't find "tailwindcss/colors".

Any ideas if this is possible?

**Comment by saadeghi** [comment, marked as answer]
Tailwind's standalone CLI (without Node.js) doesn't support third party plugins as far as I know (only includes first party plugins like Typography, etc.)

But you can directly download daisyUI's CSS file and use it in your project: 
https://unpkg.com/daisyui@2.51.5/dist/full.css

It's a large CSS file tho so you need to find a tool that removes unused CSS. In Node.js projects, there is [Purge CSS](https://purgecss.com/) you might be able to use it in other languages too but there are other tools as well. For example [this one](https://docs.rs/css-purify/latest/css_purify/) for Rust.

Let me know if you have any questions

**Reply by arqalite** [reply]
Got it - thanks!

I might end up creating a third-party tool using that css_purify crate you mentioned.

**Comment by akclace** [comment]
This build of the standalone CLI includes support for DaisyUI without using npm

https://github.com/dobicinaitis/tailwind-cli-extra
